arguments passing to `g_i_b_t`; `otherkeys` argument added to `g_i_b_t`

jemmybutton opened the pull request:
optional arguments are now passed from functions like `get_prey_of` to `get_interactions_by_taxa` using `...`;
`otherkeys` argument to `get_interactions_by_taxa` can be used for any key-value pair not listed among main function arguments in a form `otherkeys = list ("somekey" = c("value1", "value2", ...), ...`. So we can limit number of  `get_interactions_by_taxa` arguments without hampering its functionality.

jhpoelen commented:
@jemmybutton thanks for your contribution. I noticed that the tests and the documentation were not updated. Would it be an idea to add some tests and notes to ensure that your desired functionality stays around in future versions?

jemmybutton commented:
@jhpoelen Sure, sorry, i've updated documentation and added one test (if i understand correctly how it works), there's new pull request with that. There's also a new function `get_data_fields`, but it's not working, and i have no idea why (formally, it looks exactly like `get_interaction_types`, but server returns something strange).
